x/rewards/types/keys.go

```diff
@@ -18,6 +18,12 @@ const (
 
 	// prefix for the CU tracker timer store
 	MonthlyRewardsTSPrefix = "monthly-rewards-ts"
+
+	// prefix for IPRPC eligible subscriptions
+	IprpcSubscriptionPrefix = "iprpc-subscription"
```

Comment:
is this the right place for it?
compare with other keys of objects we scaffolded

Reply:
done. see https://github.com/lavanet/lava/commit/d3a07dde7f78c75958e4befa3ea4492544799f0c